.travis.yml

```diff
@@ -97,15 +97,15 @@ jobs:
         - go get -v github.com/golang/dep/cmd/dep
         - export PATH=$GOPATH/bin:$PATH
       after_success:
-        - gzip -S .darwin_amd64.gz $GOPATH/bin/hercules
+        - tar -xzvf "$GOPATH/bin/hercules.darwin_amd64.tar.gz" $GOPATH/bin/hercules LICENSE.md
```

Comment:
So this is a feature, not a bug: it is currently possible to fetch and ungzip the binary on the fly with Python. It would not be possible to achieve with tar due to the way it is implemented in Python. We are using that in a few places here and there.

I wonder if embedding the license into the binary itself can solve the issue? Otherwise, I am also fine with having two release formats, plain gzip and a tar with license.

Reply:
Ok, updated the commit accordingly. Well, having the license in the archive makes it a bit easier to repack, as it is automatically the right license of the release, also in case if it was changed in the repo.

Reply:
I tend to close this again, as the LICENSE.md anyway only includes the license of the hercules source code, but not of all the incorporated dependencies. That would maybe require something like fossa-cli to compile a complete binary distribution license file to include.